client/src/components/ContentNavigator/index.ts

```diff
@@ -247,6 +247,44 @@ class ContentNavigator implements SubscriptionProvider {
           "workbench.actions.treeView.contentdataprovider.collapseAll",
         );
       }),
+      commands.registerCommand(
+        "SAS.convertSasnbToFlw",
+        async (resource: ContentItem) => {
+          const resourceUri = getUri(resource);
+          // Ensure that the command is only available for .sasnb files
+          if (resourceUri.path.endsWith(".sasnb")) {
+            // Open window to chose the name and location of the new .flw file
+            const name = await window.showInputBox({
+              prompt: "Enter the name for the new .flw file",
+              value: resource.name.replace(".sasnb", ".flw"),
+              validateInput: fileValidator,
```

Comment:
Can we make a `flowFileValidator` here to test the file and show `InvalidFlwFileNameError` there instead of checking in the if statement below?

Reply:
done in 3bf40d5b4ae1b22e902919ff178e8e49ae207131